guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -27,52 +27,64 @@ endif::[]
 . In the {ProjectWebUI}, navigate to *Hosts* > *Register Host*.
 . Optional: Select a different *Organization*.
```

Comment:
If you want to streamline this procedure, you could combine the optional steps: "Optional: You can select a different organization, location, host group, or operating system." 

Alternatively, you could omit the optional steps if their meaning is self-evident.